pkg/log/log.go

```diff
@@ -0,0 +1,68 @@
+package log
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Level describes the current log level.
+type Level uint32
+
+const (
+	// DebugLevel log level.
+	DebugLevel Level = Level(logrus.DebugLevel)
+	// InfoLevel log level.
+	InfoLevel Level = Level(logrus.InfoLevel)
+	// ErrorLevel log level.
+	ErrorLevel Level = Level(logrus.ErrorLevel)
+)
+
+type logger struct {
+	level Level
+	entry *logrus.Entry
+	ctx   context.Context
+	err   error
+}
+
+func Info() Printer {
+	return &logger{level: InfoLevel}
+}
+
+func Error() Printer {
+	return &logger{level: ErrorLevel}
+}
+
+func Debug() Printer {
+	return &logger{level: DebugLevel}
+}
+
+// Most commonly used logging function
```

Comment:
Need to fix the comment.

And add comments for the functions above. This should have been caught by the linter.
```suggestion
// Print adds `msg` to the log at `InfoLevel`. It is a wrapper for `Info().Print(msg)`, since this is the most common use case.
```